core-interfaces/IBrowserEmulatorClass.ts

```diff
@@ -1,11 +1,10 @@
-import IUserAgent from './IUserAgent';
 import IBrowserEmulator from './IBrowserEmulator';
 
 export default interface IBrowserEmulatorClass {
   id: string;
   roundRobinPercent: number;
   engine: { browser: string; executablePath: string; revision: string };
-  new (userAgent?: IUserAgent): IBrowserEmulator;
+  new (): IBrowserEmulator;
```

Comment:
Assuming you have a way to set this in double agent?

Reply:
Yes. Each emulator has a static getter called userAgentOptions, which I override in DA (well, actually Scraper Report / Benchmarker) to select the user-agent string I want.

Reply:
This is the interface for the Class, right? Shouldn't we put it on here if you need every emulator to have it on the class?

Reply:
No because the only emulators that need it are the ones we're directly using in Scraper Report, which is our emulators. It shouldn't be required for "other developers" (if it ever comes to that... ha).

Reply:
I have a feeling we will internally forget... the interface seems just as much for us as others, but I don't have anything else to add if you don't agree.

Reply:
Ok. I added it.